NataliiaZinchenko/HW4/nzinchenko_4(converter).py

```diff
@@ -0,0 +1,8 @@
+celsius = input("Enter the temperature in Celsius: ")
```

Comment:
good job
another way:
if float(celsius) <= -273.15:
    print("Error: Temperature below absolute zero (-273.15°C)")
else:
    fahrenheit_formula = (int(celsius) * (9/5) + 32)
    print(f"{celsius}°C is equivalent to {fahrenheit_formula}°F")